tests/integrations/android_sdk/AndroidSDK/test_verify.py

```diff
@@ -210,11 +225,20 @@ def test_download_sdk(mock_tools, tmp_path):
     assert sdk.cmdline_tools_path.is_dir()
     assert sdk.cmdline_tools_version_path.is_file()
 
-    if platform.system() != "Windows":
-        # On non-Windows, ensure the unpacked binary was made executable
-        assert os.access(
-            cmdline_tools_base_path / "latest" / "bin" / "sdkmanager", os.X_OK
-        )
+    # Verify binaries marked executable for non-Windows
+    if mock_host_os != "Windows":
+        if platform.system() == "Windows":
+            # When the actual host is Windows, check the mock was called
+            mock_tools.os.access.assert_called_with(
+                cmdline_tools_base_path / "latest" / "bin" / "sdkmanager",
+                os.X_OK,
+            )
+        else:
+            # When the actual host is non-Windows, ensure the unpacked binary is executable
+            assert os.access(
+                cmdline_tools_base_path / "latest" / "bin" / "sdkmanager",
+                os.X_OK,
+            )
```

Comment:
This is the kind of test I was referring to in my comments about mocking vs platform coverage. We're explicitly testing the behaviour of Linux on Windows, and we need to insert a platform mock of `os.access` to make that work. I'm not sure that's actually gaining us anything.

Reply:
ah, yeah; I think this is another one that'll require `# pragma: no cover`. I'll verify to be sure.